backend/api/bookAVideoLinkApi.ts

```diff
@@ -0,0 +1,29 @@
+import moment from 'moment'
+
+export const bookAVideoLinkApiFactory = (client) => {
+  const processResponse = () => (response) => response.body
+
+  const post = (systemContext, url, data) => client.post(systemContext, url, data).then(processResponse())
+  const sendDelete = (systemContext, url) => client.sendDelete(systemContext, url).then(processResponse())
+
+  const matchAppointmentToVideoLinkBooking = (systemContext, { offenderNo, startTime, endTime }, { locationPrefix }) =>
+    post(systemContext, `/video-link-booking/search`, {
+      prisonerNumber: offenderNo,
+      locationKey: locationPrefix,
+      date: moment(startTime).format('YYYY-MM-DD'),
+      startTime: moment(startTime).format('HH:mm'),
+      endTime: moment(endTime).format('HH:mm'),
+    })
```

Comment:
Did this need a new endpoint in BVLS API too? Or already existing from previous work?